ui/src/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentEditor.js

```diff
@@ -10,6 +11,31 @@ export class DocumentEditor extends Component {
     error: {},
   };
 
+  get documentRequest() {
+    const request = _get(this.props, 'location.state', null);
+    if (request) {
+      const data = {
+        documentRequestPid: request.pid,
+        metadata: {
+          // NOTE: Cheating so the authors don't break our document form
+          authors: [{ full_name: request.metadata.authors, type: 'PERSON' }],
```

Comment:
Should we set the default type here? Also, since the author type is a vocabulary can we assume there is a vocabulary called `PERSON`?